gradle/libs.versions.toml

```diff
@@ -18,7 +18,7 @@ jacoco = "0.8.9"
 jandex = "3.1.0"
 jmh = "1.36"
 junit = "5.9.2"
-logback = "1.2.11"
+logback = "1.3.6"
```

Comment:
Last version is 1.4.6, is 1.4.x not compatible?

Reply:
> Last version is 1.4.6, is 1.4.x not compatible?

Requires Java 11